packages/create-waku-app/build.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+const path = require("path");
+const fs = require("fs-extra");
+
+const packageJson = require("./package.json");
+const examplesFolder = path.resolve("./examples");
+
+async function run() {
+    fs.ensureDirSync(examplesFolder);
+
+    const supportedExamples = Object.entries(packageJson.wakuExamples);
+
+    console.log("Started copying supported Waku examples.");
+
+    const copyPromises = supportedExamples.map(([name, relativePath]) => {
+        const resolvedPath = path.resolve(__dirname, relativePath);
+        const destinationPath = path.resolve(examplesFolder, name);
+        
+        return fs.copy(resolvedPath, destinationPath, { filter: nodeModulesFiler }).catch((error) => {
+            console.error(`Failed to copy example ${name} to ${destinationPath} with ${error.message}`);
+            throw Error(error.message);
+        });
+    });
+
+    await Promise.all(copyPromises)
+        .then(() => {
+            console.log("Finished copying examples.");
+        })
+        .catch((error) => {
+            console.error("Failed to copy examples due to " + error.message);
+        });
```

Comment:
We should stick to either `promise.then().catch()` syntax, or use a `try-catch` block with `async/await`, and not mix both.

My suggestion is to use the latter.